### PR TITLE
Add topics to each incident sent through kafka

### PR DESCRIPTION
**Pull request by tejido18**

### CODE REVIEW

File: src/main/java/com/uniovi/services/TopicService.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	public static List<String> getTopicsOf(Incident incident)
+	{
+		List<String> aux = new ArrayList<String>();
+		if(!incident.getLocation().equals(null))
+			aux.add("geolocated");
+		//the withOperator topics is not implemented
+		if(!incident.getAgent().getKind().equals("sensor"))
+			aux.add("sensor");
+		if(aux.size()==0)
+			aux.add("standard");
```
Comment (alejgh) [inline src/main/java/com/uniovi/services/TopicService.java]: Here you can remove the `if(aux.size()==0)`. For now every incident will have the standard topic.

Comment (alejgh) [reply src/main/java/com/uniovi/services/TopicService.java]: Regarding the "withOperator" topic, operators can be stored in the properties map of the incident, so you can do something like this:
```java
for (String key : incident.getProperties().keySet()) {
    if ("operators".equals(key)) {
        aux.add("withOperator");
        break;
    }
}
```

Comment (alejgh) [reply src/main/java/com/uniovi/services/TopicService.java]: Also, it seems like you moved every test from 'src/test' to 'src/main/test', so please put the tests back to 'src/test' if you can.

Comment (alejgh) [reply src/main/java/com/uniovi/services/TopicService.java]: And finally, codacy seems to be complaining about `if(!incident.getLocation().equals(null))`, so use `==` instead. 
The Travis build is also failing but 'm pretty sure this is not your fault, so I will make some tests to see why it is failing. For now you can work on the previous comments while I try to fix Travis.

**Review by alejgh — CHANGES_REQUESTED** [review]

**Review by alejgh — APPROVED** [review]

**Comment by tejido18** [comment]
Is it ok?

**Review by alejgh — APPROVED** [review]

**Comment by alejgh** [comment]
It's ok, sorry for the wait. You can merge this whenever you want :thumbsup:

**Comment by carmee-en** [comment]
Codecov will be patched in future pull requests. Merging branch.

**Pull request merged**